bin/release-package.php

```diff
@@ -72,8 +85,34 @@
 );
 execute( $command, 'Could not add the new package repository remote.', true, true );
 
+// Update the version numbers of each dependency of the package we are releasing.
+execute( 'bin/version-packages.sh --no-update', 'Could not update sub-package dependency versions.', true, true );
+
+// Create a new branch to prepare our release.
+$release_branch = sprintf(
+	'update/jetpack-%1$s-v%2$s',
+	$package_name,
+	$tag_version
+);
+$command        = sprintf(
+	'git checkout -b %1$s',
+	escapeshellarg( $release_branch )
+);
+execute( $command, 'Could not create new release branch.' );
+
+// Commit those changes.
+$command = sprintf(
+	'git add composer.json && ( git diff-index --quiet HEAD || git commit -m "Updating dependencies for %1$s" )',
+	escapeshellarg( $package_name )
+);
+execute( $command, 'Could not commit dependency version updates.' );
+
 // Push the contents to the package repository.
-execute( 'git push package master --force', 'Could not push to the new package repository.', true, true );
+$command = sprintf(
+	'git push package %1$s --force',
```

Comment:
do we need to use the force here, Luke?

Reply:
That's a good question. It was added in the original PR in #12475. I don't know that it's necessary since there shouldn't normally be merge conflicts when pushing new versions of the package to the package remote, but I'd love to have @tyxla's opinion on that.

Reply:
The idea here is that we might have changed something in one or more commits in the history that's shared between the jetpack repo and the package repo. Then, we'd want to propagate that change to the package, too, and overwrite any old history that lives in the package.

But yeah, as Jeremy said, normally this should be fine and no merge conflicts or history changes should be present.